### DISCUSSION TITLE
Roman sorusu programı

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus.github.io#47] [Soru Cevap]
Ali, bir romanın her gün bir önceki gün okuduğu sayfadan 5 sayfa fazlasını okumaktadır. İlk gün 10 sayfa okuyarak başlayan Alinin’ 1.000 sayfalık bir romanı kaç günde bitireceğini bulan programı kodlayınız.

**Comment by DeniZZece** [comment]
```
#include<stdio.h>
int main()
{
	int sayfa=10, toplam=0, gun=0;
	while (toplam<1000)
	{
		toplam+=sayfa;
		sayfa+=5;
		gun++;
	}
	printf("%d",gun);
}
```
